ext/semian/extconf.rb

```diff
@@ -23,7 +23,7 @@
 have_func 'rb_thread_blocking_region'
 have_func 'rb_thread_call_without_gvl'
 
-$CFLAGS = "-D_GNU_SOURCE -Werror -Wall "
+$CFLAGS = "-D_GNU_SOURCE -Wall "
```

Comment:
Switching to `ftok` will likely require changes in the external Semian API. This might not be a big deal since this branch will already require a major version bump, but using `ftok` properly will not be as simple as just replacing the `SHA1` call.

We could pass `-Wno-deprecated-declarations` to disable this *specific* warning, but I'm not sure how that will affect compilers that don't support that option.